Add Vault + AWS IRSA Integration Docs (#8647)
solo-io/gloo: docs/content/guides/integrations/vault/_index.md

---
title: Vault Integration
description: Managing secrets using HashiCorp Vault
weight: 7
---

# Overview

Gloo Edge can integrate with HashiCorp Vault to provide an alternative way to manage secrets.

Managing secrets using HashiCorp Vault has many benefits such as:

> - **Centralized Secret Management**: Vault provides a centralized solution for managing secrets which can be accessed across different environments. Only storing secrets in Kubernetes, on the other hand, gets limited so a single cluster, which may not be sufficient for complex or multi-cloud architectures.

Reviewer: ```suggestion
- **Centralized Secret Management**: Vault provides a centralized solution for managing secrets which can be accessed across different environments. When you only store secrets in Kubernetes, secret access is limited to a single cluster, which might not be sufficient for complex or multi-cloud architectures.
```
